library/kani_macros/src/lib.rs

```diff
@@ -51,6 +56,40 @@ pub fn proof(attr: TokenStream, item: TokenStream) -> TokenStream {
     // )
 }
 
+#[cfg(not(kani))]
+#[proc_macro_attribute]
+pub fn async_proof(attr: TokenStream, item: TokenStream) -> TokenStream {
+    proof(attr, item) // Treat like a normal #[kani::proof] if Kani is not active
+}
+
+#[cfg(kani)]
+#[proc_macro_attribute]
+pub fn async_proof(attr: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
This macro is more extensive than some of the others, and could probably use a comment.

Reply:
Added.